src/tabs/tab-header-bar.scss

```diff
@@ -162,7 +162,6 @@ $label-horizontal-spacing: awsui.$space-l;
 .tabs-tab-disabled {
   &,
   &:hover {
-    pointer-events: none;
```

Comment:
Does this also add pointer events to disabled tabs with no disabled reason? Is that expected?

Reply:
No, it does not, because tabs with no disabled reason has attribute `disabled`, which prevents events by default.